src/Quadmath.jl

```diff
@@ -196,15 +369,37 @@ eps(::Type{Float128}) = reinterpret(Float128, 0x3f8f_0000_0000_0000_0000_0000_00
 floatmin(::Type{Float128}) = reinterpret(Float128, 0x0001_0000_0000_0000_0000_0000_0000_0000)
 floatmax(::Type{Float128}) = reinterpret(Float128, 0x7ffe_ffff_ffff_ffff_ffff_ffff_ffff_ffff)
 
-ldexp(x::Float128, n::Cint) =
-    Float128(ccall((:ldexpq, libquadmath), Cfloat128, (Cfloat128, Cint), x, n))
+if Sys.iswindows()
+    function ldexp(x::Float128, n::Cint)
+        r = Ref{Cfloat128}()
+        ccall((:ldexpq, libquadmath),
+              Cvoid, (Ptr{Cfloat128}, Cfloat128, Cint), r, x, n)
```

Comment:
Do you know why some input args are passed by value and some by ref?

Reply:
or are they equivalent with the windows ABI?

Reply:
ah, looks like the [windows calling convention](https://docs.microsoft.com/en-us/cpp/build/x64-calling-convention?view=vs-2017) always passes 128-bit objects as pointers

Reply:
Sorry, that was just my carelessness.  I'll try again in the next commit.

Reply:
I was thinking it might be simpler to define a macro to handle this. I'm happy to look at it later this week if you like.